src/domain/transaction/stellar/stellar_transaction.rs

```diff
@@ -42,6 +47,14 @@ impl Transaction for StellarRelayerTransaction {
         &self,
         tx: TransactionRepoModel,
     ) -> Result<TransactionRepoModel, TransactionError> {
+        let signature = self
+            .signer
+            .sign_transaction(NetworkTransactionData::Stellar(
+                tx.network_data.get_stellar_transaction_data()?,
+            ))
+            .await?;
+
+        info!("signature: {:?}", signature);
```

Comment:
Should we always log this one or it was needed for developing?

Maybe to reconsider log level and add more context to log if we are going to keep it.

Reply:
It was just needed for dev